pkg/graph/resolvers/trb_request_form.go

```diff
@@ -83,17 +91,7 @@ func UpdateTRBRequestForm(
 	if systemIntakes, systemIntakesProvided := input["systemIntakes"]; systemIntakesProvided {
 		delete(input, "systemIntakes")
 
-		systemIntakeUUIDs := []uuid.UUID{}
-		if systemIntakeIFCs, ok := systemIntakes.([]interface{}); ok {
-			for _, systemIntakeIFC := range systemIntakeIFCs {
-				if systemIntakeStr, ok := systemIntakeIFC.(string); ok {
-					systemIntakeUUID, parseErr := uuid.Parse(systemIntakeStr)
-					if parseErr != nil {
-						return nil, parseErr
-					}
-					systemIntakeUUIDs = append(systemIntakeUUIDs, systemIntakeUUID)
-				}
-			}
+		if systemIntakeUUIDs, ok := systemIntakes.([]uuid.UUID); ok {
```

Comment:
Wow, I guess this is one of the benefits of having the actual type passed now! That's a lot less code!